src/evaluate.cpp

```diff
@@ -389,8 +389,8 @@ namespace {
   Score Evaluation<T>::pieces() {
 
     constexpr Color     Them = ~Us;
```

Comment:
```suggestion
    constexpr Color Them = ~Us;
```

Reply:
with the extra spaces its aligned to the variables below

Reply:
Currently, it's not aligned because the line below changed. Would you change it differently?

Reply:
A whops.. the git diff confused me a bit.. in that case I dont think theres a really nice solution..